src/cape_ros_node.cpp

```diff
@@ -65,11 +65,11 @@ std::array<uint8_t, 3> hsvToRgb(const double h, const double s, const double v)
 
 CapeRosNode::CapeRosNode(ros::NodeHandle nh)
   : nh_(nh)
-  , cos_angle_max_(std::cos(M_PI / 30))
+  , cos_angle_max_(std::cos(M_PI / 45))
```

Comment:
Since the cos_angle_max_ is read from param why change it to M_PI/45 as default instead of M_PI/30?

Reply:
I made the default value the same as in the parameters.